tests/yam/agama/login.pm

```diff
@@ -0,0 +1,23 @@
+package login;
+
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: login console
+# Maintainer: Wei Gao <wegao@suse.com>
```

Comment:
Shouldn't that line be `# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>` ?